ChangeLog.d/9126.txt

```diff
@@ -0,0 +1,6 @@
+Default behavior changes
+   * In a pure PSA client build (i.e. MBEDTLS_PSA_CRYPTO_CLIENT &&
```

Comment:
Minor: “pure PSA client build” is hard to parse: is it a (pure PSA) (client build)? a pure (PSA client build)? a (pure (PSA client)) build? What does this mean anyway: `MBEDTLS_PSA_CRYPTO_CLIENT` and nothing else?

I suggest calling it a “PSA client-only build”. (Or should this be hyphenated as “PSA-client-only build”?)